### PR TITLE
frontend structure done

### PR DESCRIPTION
**Pull request by guanhchan**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="App-header">
+        <Header></Header>
```
Comment (SunkissedQueen) [inline src/App.js]: component calls should look like a self-closing html tag
`<Header />`

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/index" element={<MemeIndex />} />
+          <Route path="/show" element={<MemeShow />} />
+          <Route path="/new" element={<MemeNew />} />
+          <Route path="/edit" element={<MemeEdit />} />
+          <Route path="*" element={<NotFound />} />
+        </Routes>
+
+        <Footer></Footer>
```
Comment (SunkissedQueen) [inline src/App.js]: component calls should look like a self-closing html tag
`<Footer />`

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+import { Navbar } from 'reactstrap';
```
Comment (SunkissedQueen) [inline src/components/Footer.js]: when importing multiple components from the same location encapsulate them within parenthesis and separate with a comma
`import { Nav, Navbar } from 'reactstrap'`

File: src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    UncontrolledDropdown,
+    DropdownToggle,
+    DropdownMenu,
+    DropdownItem,
+    NavbarText,
+} from 'reactstrap';
+
+import { useState } from 'react';
+
+const Header = (args) => {
```
Comment (SunkissedQueen) [inline src/components/Header.js]: Please share a brief explanation for the `args` parameter on line 18 and 28. What data is being transferred?

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Some minor changes. Please correct the component calls and imports. 

Trying to understand the flow data using the `args`. Please give a brief explanation.

File: src/components/Header.js (hunk 4 above)
Comment (guanhchan) [reply src/components/Header.js]: args, was removed, nothing was being passed yet.

**Comment by guanhchan** [comment]
I made the changes, thank you for the feedback

**Review by SunkissedQueen — APPROVED** [review]
Thank you for the update.

**Pull request merged**